docs/learn/smart-contract-internals/debugging-errors.mdx

```diff
@@ -0,0 +1,117 @@
+---
+sidebar_position: 70
+title: Debugging Errors
+description: Debug Soroban Errors.
+---
+
+<head>
+  <title>Debug smart contracts natively and as WASM.</title>
+  <meta charSet="utf-8" />
+  <meta
+    property="og:title"
+    content="Debug smart contracts natively and as WASM."
+  />
+  <meta
+    property="og:description"
+    content="Compile smart contracts natively for local testing or into WASM for deployment in order to debug."
+  />
+</head>
```

Comment:
This looks like it's duplicated from other page, so it may need to be changed. Or, we can add a more detailed description on line 4, and this whole thing could be removed. Docusaurus already generates these `<meta>` tags in the document head, based on the "frontmatter" present in the file.